CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 0.24.0 / 2018-10-11
+
+This release has a breaking changes for `prometheus_operator_.*` metrics.
+
+`prometheus_operator_alertmanager_reconcile_errors_total` and `prometheus_operator_prometheus_reconcile_errors_total`
+are now combined and called `prometheus_operator_reconcile_errors_total`.
+Instead the metric has a "controller" label which indicates the errors from the Prometheus or Alertmanager controller.
+
+The same happened with `prometheus_operator_alertmanager_spec_replicas` and `prometheus_operator_prometheus_spec_replicas`
+which is now called `prometheus_operator_spec_replicas` and also has the "controller" label.
+
+The `prometheus_operator_triggered_total` metric now has a "controller" label as well and finally instruments the
+Alertmanager controller.
+
+For a full description see: https://github.com/coreos/prometheus-operator/pull/1984#issue-221139702
+
+* [CHANGE] Default to Node Exporter v0.16.0 (#1812)
+* [CHANGE] Update to Go 1.11 (#1855)
+* [CHANGE] Default to Prometheus v2.4.3 (#1929) (#1983)
+* [CHANGE] Default to Thanos v0.1.0 (#1954)
+* [CHANGE] Overhaul metrics while adding triggerBy metric for Alertmanager (#1984)
+* [FEATURE] Add multi namespace support (#1813)
```

Comment:
we should add a note that says that this introduces a breaking change. The operator no longer accepts the `--namespace` flag but rather `--namespaces`

Reply:
@metalmatze can you follow up with a fix to `release-0.24` branch and merge that back into `master` once its merged?